src/schemas/686c-674/schema.js

```diff
@@ -20,15 +20,24 @@ const schema = {
     genericLocation: {
       type: 'object',
       properties: {
-        state: {
+        isOutsideUS: {
+          type: 'boolean',
+          default: false,
+        },
+        country: {
           type: 'string',
           maxLength: 30,
-          pattern: textOnlyPattern,
+          pattern: '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*0-9]+$',
```

Comment:
I've seen this pattern before, but am still a little unclear on what exactly it checks for O.o